astroquery/esa/euclid/core.py

```diff
@@ -116,6 +117,156 @@ def __init__(self, *, environment='PDR', tap_plus_conn_handler=None, datalink_ha
         if show_server_messages:
             self.get_status_messages()
 
+    def cross_match_basic(self, *, table_a_full_qualified_name, table_a_column_ra, table_a_column_dec,
+                          table_b_full_qualified_name='', table_b_column_ra='', table_b_column_dec='',
```

Comment:
We have default table names for gaia, would any of that make sense here, too?
If not, then I would still use the `None` as default rather than null strings.